build.rs

```diff
@@ -4102,14 +4103,6 @@ mod c {
             ],
         );
 
-        // When compiling in rustbuild (the rust-lang/rust repo) this library
-        // also needs to satisfy intrinsics that jemalloc or C in general may
-        // need, so include a few more that aren't typically needed by
-        // LLVM/Rust.
-        sources.extend(&[
-            "ffsdi2.c",
```

Comment:
How come this was moved up?

Reply:
I could leave it where it is, but that invites questions (why is it separate from the rest of the files?).

The comment here suggests an answer, but doesn't make sense given that the file's inclusion is unconditional (the condition was removed in 91eaa859027ea4806997112cb9f53b25e51106ca, but I don't know why).

So since the inclusion is unconditional, I figured I'd just move it into the main slice.

Reply:
This is currently in a dedicated section because of the comment. This file is not needed to compile Rust but it's needed to compile Rust's dependencies, namely jemalloc.

Reply:
OK, so the conditional bit should be restored?